src/migrations/1692699242541-add-is-favorite-column.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class addIsFavoriteColumn1692699242541 implements MigrationInterface {
+  name = 'addIsFavoriteColumn1692699242541';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE \`private_name_tag\` ADD \`is_favorite\` tinyint NOT NULL DEFAULT 0`,
+    );
+  }
+
+  public async down(_queryRunner: QueryRunner): Promise<void> {}
```

Comment:
chỗ này rollback được thì mình nên viết down cho nó ạ
```suggestion
  public async down(_queryRunner: QueryRunner): Promise<void> {
      await queryRunner.query(
      `ALTER TABLE \`private_name_tag\` DROP COLUMN \`is_favorite\``,
    );
  }
```